examples/sveltekit-server/.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+	"useTabs": true,
```

Comment:
Is this the default for sveltekit projects?

Reply:
haha yes https://github.com/sveltejs/kit/blob/master/.prettierrc#L2C3-L2C10